README.md

```diff
@@ -751,7 +751,7 @@ Synonyms: autocomplete, search as you type, suggestions
 * [Algolia](https://www.algolia.com/)
 * Vespa
 * Elastic
-* Solr
+* [Solr](https://solr.apache.org/) - OSS Elasticsearch alternative
```

Comment:
I don't know that I would refer to Solr as an "Elasticseach alternative"....   The term "alternative" is fairly ambgious, and OpenSearch is actually a much closer alternative.   I'd rather hear about what makes it unique... 

The http://solr.apache.org website describes it as:

> Solr is the blazing-fast, open source, multi-modal search platform built on the full-text, vector, and geospatial search capabilities of Apache Lucene

Reply:
Gotcha 👍🏽  Do you want me to write anything else, or do you want only the link for the PR?

Reply:
it's maybe a bit wordy, but could literally say "Solr is the blazing-fast, open source, multi-modal search platform built on the full-text, vector, and geospatial search capabilities of Apache Lucene"